data/json/martialarts.json

```diff
@@ -936,6 +936,77 @@
             "tec_venom_snake_bite",
             "tec_venom_snake_strike"
         ]
+    },{
+        "type" : "martial_art",
+        "id" : "style_hema",
+        "name" : "HEMA",
+        "description": "The modern recreation of European fencing, based off the manuals of Medieval and Renaissance fencing masters. It emphasises quick footwork and aggressive striking. Skilled fencers can apply their techniques to make lightning fast attacks and devastating combos, while maintaining their defense.", 
```

Comment:
Should probably spell out Historic European Martial Art (or whatever HEMA is supposed to represent) here so folks don't constantly ask.